### DISCUSSION TITLE
A few quick questions before I install PMM

### DISCUSSION THREAD

**Discussion by dr-doom0069** [meisnate12/Plex-Meta-Manager#1733] [Q&A]
I have a pretty typical Media server. PLEX docker running in Unraid. Lifetime Plex pass, IGPU 770 transcode to RAM, Meta data is on SSD. 95% of its use if for watching at home on LAN, 5% is for remote at work. I have about 60TB of media. Nvidia Shields as clients.

Plex has been running great for a long time. I have no playback Issues. 

I do have a few things that I would like to change and I finally decided to look into PMM to see if it can help. I'm more of a follow a tutorial type of guy when it comes to scripts and more complicated stuff so I figured I get this all figured out before I decide to run PMM.

1. I have a large Stand Up Comedy library. The ["Library View"](https://imgur.com/Ds2D02q) is kind of a mess, most titles start with the name of the comic, but not all do so I have to manually edit them after adding them to the library. There are also about 300 titles, so it can be a bit disorganized to scroll through. My solution to this is to create [Collections for each Comic](https://imgur.com/2AX9qt3). This works great. My problem is that I have to manually update the collections any time I add new media. This is a pain... **Can PMM update these Collections as a scheduled task based on the comic name in the file?**

2. I also have a large [Documentary library](https://imgur.com/K0wxTgn). I Use [Colima](https://github.com/defract/Colima.bundle) to merge Movies and TV Shows into one Library. This works fine, but its not perfect. It displays everything as [a tv show](https://imgur.com/pr0Ue36) (not a big deal, but it does add one extra click). Also it occasionally doesn't keep the correct episode or time you left off of an episode... **Can PMM merge Movie and TV Show libraries? or can it make the Movies display as movies again if I still use Colima?**

3. In A TV Library can I change the ["Recently Released Episodes"](https://imgur.com/l5Yrdvo) to just "Recently Released". I don't need to see all 10 episodes that just released at once taking up the whole row. I just want to see one Tile for the whole Show. Similar to "Recently Added" 

4. Can I remove things like "Rediscover" "More from HBO" "Top Rated TV" etc. from the TV Recommended tab?

5. Say I create a [TV Collection](https://imgur.com/Zw1zRYW) with PMM called ["Network TV"](https://imgur.com/ymLjuVl) can I set parameters so it will include only the networks I want in that specific collection?  

6. Can I manually Edit PMM Collections from the Plex GUI? If I do, will the edits I make change the next time PMM runs? For example the TV show "Arrested Development" was on FOX and Netflix. I would like to edit the Metadata to only display on one collection. Or the Networks FX and FXX, I would just like to have one collection titled ["FX"](https://imgur.com/uba7J8O)

7. Last question. Should I remove all the Collections I created in Plex before running PMM for the first time?

Thanks in advance.

**Comment by JohnFawkes** [comment]
1. Yes
2. No it likely will not work with colima and PMM can not merge libraries
3. No
4. No
5. Yes
6. No if you edit in the UI, PMM will just revert it. Makes the changes in PMM instead
7. No you do not have to do that. The only issue that may happen is if collections you made are named the same as a PMM collection, you could get dupe collection warnings

**Reply by chazlarson** [reply]
The caveat on #1 would be that you would have to define the comic names in advance.  PMM can filter on the file path, but it can't _discover_ things that way.  For example, if you create a folder for some new comic, you will have to touch the PMM config to add the collection for that person, but future things that get added to that folder will get added to the collection without manual attention.
